MediaBrowserTv/app/src/main/java/tv/emby/embyatv/presentation/MyImageCardView.java

```diff
@@ -64,21 +64,21 @@ public MyImageCardView(Context context, AttributeSet attrs, int defStyle, boolea
         }
 
         LayoutInflater inflater = LayoutInflater.from(context);
-        View v = inflater.inflate(tv.emby.embyatv.R.layout.image_card_view, this);
-
-        mImageView = (ImageView) v.findViewById(tv.emby.embyatv.R.id.main_image);
-        mInfoArea = v.findViewById(tv.emby.embyatv.R.id.info_field);
-        mTitleView = (TextView) v.findViewById(tv.emby.embyatv.R.id.title_text);
-        mContentView = (TextView) v.findViewById(tv.emby.embyatv.R.id.content_text);
-        mBadgeImage = (ImageView) v.findViewById(tv.emby.embyatv.R.id.extra_badge);
-        mOverlayName = (TextView) v.findViewById(tv.emby.embyatv.R.id.overlay_text);
+        View v = inflater.inflate(org.jellyfin.androidtv.R.layout.image_card_view, this);
```

Comment:
This should be removed entirely and the resources should be loaded at the beginning of the file with the line `import org.jellyfin.androidtv.R` instead.

Reply:
As in move this stuff up? Such that the top of the file looks like:

```
import org.jellyfin.androidtv.R

View v = inflater.inflate(R.layout.image_card_view, this);

mImageView = (ImageView) v.findViewById(R.id.main_image);
mInfoArea = v.findViewById(R.id.info_field);
mTitleView = (TextView) v.findViewById(R.id.title_text);
mContentView = (TextView) v.findViewById(R.id.content_text);
mBadgeImage = (ImageView) v.findViewById(R.id.extra_badge);
mOverlayName = (TextView) v.findViewById(R.id.overlay_text);

```

Reply:
Ah no I wasn't clear enough, just add R to the import section at the beginning of the file so you don't even need to append the package name here.

Reply:
Ah right, just throw the import at the top and change all the lower lines to using `R.` sounds cleaner.